app/R/emtr.R

```diff
@@ -0,0 +1,572 @@
+parameters_from_file <- function(parameters_file) {
```

Comment:
I don't think you use this function anywhere, you could change it back to the original here.